app/src/validation/post.js

```diff
@@ -22,10 +22,34 @@ const validatePost =async function (post) {
     if( await postModel.isSlugExist(post)) {
         error.push('The slug already exists');
     }
-   
+    let sign = post.slug;
+    if (hasForbiddenSigns(sign)) {
+        error.push('Slug can not contain polish signs'); 
+    }
+
     return error;
 };
+/**
+ * @param {string} post 
+ * @returns {boolean} 
+ */
+function hasForbiddenSigns(sign) {
+
+    if (typeof sign == 'string') {
```

Comment:
Spodziewasz się tutaj string. Jeżeli to nie jest ten typ i koniecznie chcesz to sprawdzać to po prostu rzuć wyjątek na początku funkcji, a resztę zrefaktoruj tak żeby była czytelna

Reply:
Nie wiem jak miałby wyglądać taki wyjątek dlatego proponuję inaczej, może przejdzie.